### PR TITLE
Feature/infogrid

### PR DESCRIPTION
**Pull request by knees4bees**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ ] Fix
- [x] Feature
- [ ] Refactor
- [x] Style
- [ ] Testing

### Are there any known bugs?

- [ ] No
- [x] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Adds styling and calculations for planet detail page

### What ticket(s) do the changes resolve?
Partial resolution of #39
Please Link Issues to this PR as necessary.

### Notes for the Reviewer
This is a work in progress. Still need to finish styling the distance and length of day sections but pushing now so we can get some user feedback.

### How to Test Changes?
Check info displays for all planets. Earth has special formatting, and sometimes the wording changes depending on whether the planet's numbers are smaller or larger than those for Earth.

### Next Iterations
Finish styling distance and length of day sections.

### CODE REVIEW

**Review by ConnorAndersonLarson — APPROVED** [review]
Looks good, just need to fix that NaN bug in the view (which is already being worked on)

File: src/components/Planetarium/Planetarium.tsx [hunk 1]
```diff
@@ -2,25 +2,36 @@ import React from 'react';
...
+      formatData = `${data}`;
     } else if (key === 'gravity') {
       descriptor = 'm/s²'
+      formatData = `${data}`;
     }
-    return (`${data} ${descriptor}`)
+    return (`${formatData} ${descriptor}`)
   }
 
   const planetCards = allPlanets.map(planet => {
```
Comment (alexthompson207) [inline src/components/Planetarium/Planetarium.tsx:37]: This is cleaner this way!

File: src/utilities.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const formatLargeNumbers = (number: number): string => {
+  let totalFormatted = number.toString().split('.');
+  totalFormatted[0] = totalFormatted[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  return (totalFormatted.join('.'));
+}
```
Comment (alexthompson207) [inline src/utilities.tsx:5]: I like moving this into a separate file!

**Review by alexthompson207 — APPROVED** [review]
I love what you've done so far, the UI is soooooo much better and kid friendly this way!!!

**Pull request merged**